javascript/javascript1/week2/class07Students.js

```diff
@@ -0,0 +1,52 @@
+// copy your homework solution here to make the webpage work!
+
+const class07Students = [];
+let count = 0;
+function addStudentToClass(studentName) {
+   if (studentName.toLowerCase() == 'queen') {
+class07Students.push(studentName);
+  }
+  else if ((class07Students.indexOf(studentName) !== -1) || (studentName == '' ) ){
+    return 'Cannot add same students to class 07';
+ }
+  else if (class07Students.length >=6 ) {
+    return 'Cannot add more students to class 07';
+ }
+  else {
+    class07Students.push(studentName);
+  return 'Successfully added $studentName to the class';
+  }
+}
+
+
+function getNumberOfStudents() {
+    if (class07Students.length < 6) {
+    return class07Students.length;
+}
+  else
+  {
+    return 'Cannot add more students to class 07';
+  }
+}
```

Comment:
Since this method is called `getNumberOfStudents` I would only expect it to return the `.length` part, not the 'Cannot add more students to class 07'.

Reply:
This means this function could simply be:
```javascript
function getNumberOfStudents() {
  return class07Students.length
}
```